util/security/jwtutil.py

```diff
@@ -106,7 +104,11 @@ def decode(jwt, key="", verify=True, algorithms=None, options=None, **kwargs):
             "Algorithms `%s` are not whitelisted. Allowed: %s" % (algorithms, ALGORITHM_WHITELIST)
         )
 
-    return _StrictJWT().decode(jwt, key, verify, algorithms, options, **kwargs)
+    # verify is a legacy option in PyJWT, should be moved to options as verify_signature
+    if (not options and verify) or "verify_signature" not in options:
```

Comment:
can we simplify this condition? would it behave the same to always set `options["verify_signature"] = verify` *unless* the key is already in the `options` dictionary?
so something like:
```
if "verify_signature" not in options:
    options["verify_signature"] = verify
```

Reply:
yep, that's pretty close to the change I just pushed up. Need to check if `options` is none first, but this should do it.